samples/snippets/create_kmeans_model_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def test_kmeans_sample():
+# [START bigquery_dataframes_bqml_kmeans]
+    import bigframes.pandas as bpd
+    import bigframes
+    from bigframes import dataframe
+    import bigframes.pandas as bpd
+    import datetime
+
+    #Load data from BigQuery
```

Comment:
Comments should explain why, not what the code is doing. go/comments

That said, we're in the unique position of writing a code sample, so a bit of explanation could be useful. "Load data from BigQuery" is not sufficient, though. Explain what the user needs to do and why.